jina/types/document/__init__.py

```diff
@@ -506,18 +538,61 @@ def get_attrs(self, *args) -> Dict[str, Any]:
                 ret[k] = None
         return ret
 
+    def get_attrs_values(self, *args) -> List[Any]:
```

Comment:
Personally, I would prefer `def get_attrs_values(self, attributes: List[str]) -> Dict[Any]:` instead of using the magic `*args`.

And as a return value rather a Dict, to not rely on the order. But that might also be stupid, depending, where this function is usually called.

Reply:
there is a version that does this (returning a Dict), this one is intended to extract directly the values in one shot by the drivers to have it ready for batching. In the same spirit as `documentSet.all_embeddings`